.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+
+language: go
+go:
+- 1.5
```

Comment:
Just tested it and they can all be built. Do we want to support more than those?